packages/react-native-editor/ios/.xcode.env

```diff
@@ -0,0 +1 @@
+export NODE_BINARY=$(command -v node)
```

Comment:
This file is being consistently added whenever I run `pod install`, though it does not appear in the RN upgrade helper. Interested to hear thoughts on including it.

Reply:
I believe it should be committed, and individuals can create a `.xcode.env.local` to modify the configuration if necessary for their individual local environment. Its exclusion was likely an oversight from prior upgrade work. 

We should [add `.xcode.env.local`](https://github.com/facebook/react-native/pull/33546/files#diff-7f6c5fe9b71f5557bef56937bf5bf40b0a8a7b19410337b4419bc1c9e3a0d284) to this project's `.gitignore` configuration.

Reply:
Thanks for the confidence check, David! I just found that `.xcode.env.local` is already in `packages/react-native-editor/.gitignore`, so it seems we're all set on that front:

https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/.gitignore#L64